.gitignore

```diff
@@ -3,3 +3,4 @@
 /.phpunit.result.cache
 /phpunit.xml
 /phpstan.neon
+/.idea
```

Comment:
Please, remove this string. If you need to ignore your preferred IDE, use [local `.gitignore`](https://stackoverflow.com/a/1753078).